### PR TITLE
Throw if mcData is invalid

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

File: src/transforms/serializer.js [hunk 1]
```diff
@@ -15,8 +15,12 @@ const protocols = {}
 function createProtocol (state, direction, version, customPackets, compiled = true) {
   const key = state + ';' + direction + ';' + version + (compiled ? ';c' : '')
   if (protocols[key]) { return protocols[key] }
+  
   const mcData = require('minecraft-data')(version)
-
+  if (mcData === null) {
+    throw new Error(`minecraft-data is null for version ${version}`)
```
Comment (rom1504) [inline src/transforms/serializer.js:21]: I think the error should be something like "no data available for version X"

Comment (extremeheat) [reply src/transforms/serializer.js:21]: also the `=== null` won't match if it returns undefined, a simple if would work

Comment (rom1504) [reply src/transforms/serializer.js:21]: Minecraft data returns null that's fine https://github.com/PrismarineJS/node-minecraft-data/blob/master/index.js#L33

Better not rely on implicit cast if we can

**Review by rom1504 — APPROVED** [review]

**Comment by rom1504** [comment]
You need to run npm run fix

**Comment by u9g** [comment]
> You need to run npm run fix

fixed :D

**Review by rom1504 — APPROVED** [review]

**Pull request merged**